support/cas-server-support-rest-core/src/main/java/org/apereo/cas/support/rest/resources/RestResourceUtils.java

```diff
@@ -39,8 +39,12 @@
             .findAndRegisterModules()
             .configure(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE, false)
             .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
-            .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
-            .enableDefaultTyping(ObjectMapper.DefaultTyping.NON_FINAL, JsonTypeInfo.As.PROPERTY);
+            .setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
+        MAPPER.activateDefaultTyping(
+                MAPPER.getPolymorphicTypeValidator(),
+                ObjectMapper.DefaultTyping.NON_FINAL,
+                JsonTypeInfo.As.PROPERTY
```

Comment:
Why this change?

Reply:
enableDefaultTyping is deprecated and sonar check points to this https://rules.sonarsource.com/java/RSPEC-4544

Reply:
under the hood jackson is doing the same in enableDefaultTyping.

Reply:
Indeed. However, since that is unrelated to your changeset, please reset back to original. If you want to clean up deprecated instances such as this, that would be a separate PR and one that needs to be applied everywhere to be consistent.